docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs

```diff
@@ -10,39 +10,49 @@ public static class LbfgsMaximumEntropy
     {
         public static void Example()
         {
-            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
-            // as a catalog of available operations and as the source of randomness.
-            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            // Create a new context for ML.NET operations. It can be used for
+            // exception tracking and logging, as a catalog of available operations
+            // and as the source of randomness. Setting the seed to a fixed number
+            // in this example to make outputs deterministic.
             var mlContext = new MLContext(seed: 0);
 
             // Create a list of training data points.
             var dataPoints = GenerateRandomDataPoints(1000);
 
-            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            // Convert the list of data points to an IDataView object, which is
+            // consumable by ML.NET API.
             var trainingData = mlContext.Data.LoadFromEnumerable(dataPoints);
 
             // Define the trainer.
             var pipeline =
-                    // Convert the string labels into key types.
-                    mlContext.Transforms.Conversion.MapValueToKey(nameof(DataPoint.Label))
-                    // Apply LbfgsMaximumEntropy multiclass trainer.
-                    .Append(mlContext.MulticlassClassification.Trainers.LbfgsMaximumEntropy());
+            // Convert the string labels into key types.
+                mlContext.Transforms.Conversion
+                .MapValueToKey(nameof(DataPoint.Label))
+            // Apply LbfgsMaximumEntropy multiclass trainer.
+                .Append(mlContext.MulticlassClassification.Trainers
+                .LbfgsMaximumEntropy());
```

Comment:
1. Wrap the entire argument if possible (before `mlContext`)
2. Indent this line an additional 4 spaces (8 total from current location)